distribution/docs/src/main/resources/_contents/config.adoc

```diff
@@ -1,22 +1,28 @@
-Version ${version}. Copyright (c) Codice Foundation
+Version ${project.version}. Copyright (c) Codice Foundation
 :imagesdir: target/doc-contents/images
 :toc: left
-:icons: font
+:toclevels: 3
 :example-caption:
 :source-highlighter: coderay
 :sectlinks:
 :sectanchors:
+:icons: font
 :data-uri:
 :sectnums:
+:branding: DDF
+:adoc-include: ${project.build.directory}/doc-contents/_contents
 :title-logo: logo_ddf.png
 :last-update-label: Copyright (c) Codice Foundation. +
 This work is licensed under a Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0). +
 This page last updated:
 
 ifdef::backend-pdf[]
-== License
+[colophon]
+= License
+Copyright (c) Codice Foundation. +
 This work is licensed under a http://creativecommons.org/licenses/by/4.0[Creative Commons Attribution 4.0 International License].
+<<<
```

Comment:
It got mangled in a rebase. The `<<<` is supposed to trigger a page break after the colophon/licensing statement. Fixed now.